libraries/disp3D/engine/model/items/digitizer/digitizersettreeitem.cpp

```diff
@@ -241,6 +241,11 @@ void DigitizerSetTreeItem::addData(const FIFFLIB::FiffDigPointSet& tDigitizer, Q
     for(int i = 0; i < itemList.size(); ++i) {
         DigitizerTreeItem* item = dynamic_cast<DigitizerTreeItem*>(itemList.at(i));
 
+        if(item == NULL || item == nullptr || item == Q_NULLPTR){
```

Comment:
How about `if(item == Q_NULLPTR){` ? Do we always need to check for all three? Q_NULLPTR should either be null or nullptr depending on whether C++11 flags are set or not.